src/Microsoft.ML.Transforms/HashJoiningTransform.cs

```diff
@@ -541,28 +540,11 @@ private ValueGetter<VBuffer<uint>> ComposeGetterVecToVec<TSrc>(Row input, int ii
             var mask = (1U << _exes[iinfo].HashBits) - 1;
             var hashSeed = _exes[iinfo].HashSeed;
             bool ordered = _exes[iinfo].Ordered;
-            TSrc[] denseValues = null;
             return
                 (ref VBuffer<uint> dst) =>
                 {
                     getSrc(ref src);
                     Host.Check(src.Length == expectedSrcLength);
-                    ReadOnlySpan<TSrc> values;
```

Comment:
>ReadOnlySpan<TSrc> values; [](start = 20, length = 26)

I have no idea why, but as soon as I add ReadOnlySpan inside delegate, I start getting BadImage exception.
So I access elements directly instead of that strange copying mechanism.

Reply:
ouch. Is this something .NET colleagues should look at? It kind of smells of a .NET issue to me

---
In reply to: [242317331](https://github.com/dotnet/machinelearning/pull/1892#discussion_r242317331) [](ancestors = 242317331)

Reply:
@danmosemsft do you know who can take a look on this?